oak-blob-cloud-azure/src/test/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzuriteDockerRule.java

```diff
@@ -36,23 +37,30 @@
 
 public class AzuriteDockerRule implements TestRule {
 
-    private static final String IMAGE = "mcr.microsoft.com/azure-storage/azurite:3.15.0";
-    
+    private static final String IMAGE = "mcr.microsoft.com/azure-storage/azurite:3.19.0";
+
     public static final String ACCOUNT_KEY = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
     public static final String ACCOUNT_NAME = "devstoreaccount1";
 
     private static final String CONTAINER_SUFFIX = UUID.randomUUID().toString().substring(0, 8);
 
     private final DockerRule wrappedRule;
 
+    private static final AtomicReference<Exception> STARTUP_EXCEPTION = new AtomicReference<>();
+
     public AzuriteDockerRule() {
         wrappedRule = new DockerRule(ImmutableDockerConfig.builder()
             .image(IMAGE)
             .name("oak-test-azurite-" + CONTAINER_SUFFIX)
             .ports("10000")
             .addStartedListener(container -> {
-                container.waitForPort("10000/tcp");
-                container.waitForLog("Azurite Blob service is successfully listening at http://0.0.0.0:10000");
+                try {
+                    container.waitForPort("10000/tcp");
```

Comment:
Would it make sense to increase the timeout in `waitForPort()` to prevent tests to be ignored too often, if the origin of the issue is slow Jenkins nodes? The default timeout is 30s and it could be increased to 60s, for example .